TestCases/sliding_interface/uniform_flow/uniform_SM.cfg

```diff
@@ -0,0 +1,267 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
We have up to 75 minutes now for the regression tests, and we're usually around 50 minutes, so I think adding a few more test cases would be good. We'll ask the Travis CI folks for more help if we need it, and I'd rather we have good regression test coverage.